pkg/namesgenerator/names-generator.go

```diff
@@ -364,6 +373,9 @@ var (
 		// Srinivasa Ramanujan - Indian mathematician and autodidact who made extraordinary contributions to mathematical analysis, number theory, infinite series, and continued fractions. - https://en.wikipedia.org/wiki/Srinivasa_Ramanujan
 		"ramanujan",
 
+		// Sally Kristen Ride was an American physicist and astronaut. https://en.wikipedia.org/wiki/Sally_Ride
```

Comment:
You can also add she was the first American woman in space, and the youngest American astronaut.